README.adoc

```diff
@@ -1,39 +1,78 @@
-image::https://cdn.rawgit.com/decidim/decidim/develop/logo.svg[Decidim Logo,400]
-
-The participatory democracy framework.
-
-____
-Free Open-Source participatory democracy, citizen participation and open government for cities and organizations
-____
-
-https://decidim.org[Decidim] is a participatory democracy framework, written in Ruby on Rails, originally developed for the Barcelona City government online and offline participation website.
-Installing these libraries will provide you a generator and gems to help you develop web applications like the ones found on <<example-applications,example applications>> or like https://try.decidim.org[our demo application].
-
-All members of the Decidim community agree with http://www.decidim.org/contract/[Decidim Social Contract or Code of Democratic Guarantees].
+:uri-blog: https://decidim.org/blog
+:uri-chat: http://chat.decidim.org
+:uri-contributing: xref:CONTRIBUTING.adoc
+:uri-demo: https://try.decidim.org
+:uri-docs-authorizations: https://docs.decidim.org/en/customize/authorizations/
+:uri-docs-example-applications: https://docs.decidim.org/en/develop/guide_example_apps/
+:uri-docs-getting-started: https://docs.decidim.org/en/install/
+:uri-docs: https://docs.decidim.org/
+:uri-github-dependents: https://github.com/decidim/decidim/network/dependents?type=application
+:uri-metadecidim: https://meta.decidim.org
+:uri-modules: https://decidim.org/modules
+:uri-opencollective-members: https://opencollective.com/decidim/contribute/member-39953
+:uri-opencollective-partners: https://opencollective.com/decidim/contribute/partner-33556
+:uri-opencollective: https://opencollective.com/decidim
+:uri-producing-oss-license: http://producingoss.com/en/governments-and-open-source.html#starting-open-for-govs
+:uri-propose-new-features: https://meta.decidim.org/processes/roadmap
+:uri-releases: https://github.com/decidim/decidim/releases
```

Comment:
This is linked with the label "Our latest release".

Maybe we should link it to this URL instead?
https://github.com/decidim/decidim/releases/latest

Reply:
I prefer to not show only the last one, but all the releases, as the last one for instance is a patch version, and only has bug fixes and that's not something that newcomers could find interesting IMHO. So, I prefer to leave this URL, and fix the label to "Our latest releases".  Fixed on 651dc1f659a1b4dcc83e3a01bd58aaee277087c4

Reply:
Works for me!